sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -83,7 +74,7 @@ export class MessageSender extends LinkEntity {
    * @property {Sender} [_sender] The AMQP sender link.
```

Comment:
I think we should be able to update this to 
```suggestion
    * The AMQP sender link.
```
The other tags were needed when the code was JS, but now that we're using TS our doc tools can infer this.